Automattic/woocommerce-payments
# Add support for kanji and kana statement descriptors

Fixes #6874 

#### Changes proposed in this Pull Request
For JP Stripe accounts, add statement descriptors for kana and kanji. 
Under the latin statement descriptor, if it's a JP stripe account, we need to show another 2 inputs for kana and kanji statement descriptors.
Max length for kanji - 17 chars
Max length for kana - 22 chars
<img src="https://user-images.githubusercontent.com/5121465/261484253-671d5407-fc3f-4001-91c0-d232936e9865.png" />








#### Testing instructions





* checkout server PR 3801-gh-Automattic/woocommerce-payments-server 
* use a JP Stripe account
* go to Woo->Settings->Payments
* check `Transactions` section, the inputs for kana and kanji statement descriptors should be present under the latin statement descriptor input
* check for non JP Stripe account. the inputs should not be shown.

-------------------


**Post merge**
